docs/configs/site.js

```diff
@@ -45,7 +45,7 @@ export default {
         children: [
           {
             key: 'docs0',
-            text: 'latest(3.0.0)',
+            text: 'latest(3.1.0)',
```

Comment:
I'm not sure is it OK?

Reply:
I think we should better show both version 3.0.1 and 3.1.0